Meteor/main.yaml

```diff
@@ -0,0 +1,8 @@
+sources:
+  cloudradar:
```

Comment:
We tried to use the instrument identifier as the second level in the intake hierarchy. The dataset identifies the `instrument_id` as `LIMRAD94`. Thus I would recommend to change this identifier to `LIMRAD94` as well.